tools/generators/version-bump/index.ts

```diff
@@ -26,6 +26,12 @@ function runMigrationOnProject(host: Tree, schema: ValidatedSchema, userLog: Use
   const packageJsonPath = options.paths.packageJson;
   let nextVersion = '';
 
+  console.log(schema.name, schema.exclude);
```

Comment:
Leftover from testing? (if it's intended it should probably be a little more informative)